gammapy/scripts/cta_irf.py

```diff
@@ -0,0 +1,105 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+import astropy.units as u
+from astropy.table import Table
+from ..utils.fits import get_hdu
+from ..irf import EffectiveAreaTable2D
+from ..background import Cube
+from ..irf import EnergyDispersion2D
+from ..irf import EnergyDependentMultiGaussPSF
+
+__all__ = [
+    'CTAIrf',
+]
+
+
+class CTAIrf(object):
+    """Contain CTA instrument response functions.
+
+    Class handling CTA instrument response function. For now
+    we use the production 2 of the CTA IRF
+    (https://portal.cta-observatory.org/Pages/CTA-Performance.aspx)
+    adapted from the ctools
+    (http://cta.irap.omp.eu/ctools/user_manual/getting_started/response.html).
+    The IRF format should be complient compliant with the one discussed
+    at http://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/.
+    Waiting for a new public production of the CTA IRF,
+    we'll fix the missing pieces.
+
```

Comment:
You could leave as a note or TODO a link to
https://gammapy.readthedocs.io/en/latest/api/gammapy.data.DataStoreObservation.html

Maybe something like this:
```
This class is similar to `~gammapy.data.DataStoreObservation`,
but only contains IRFs (no event data or livetime info).
TODO: maybe re-factor code somehow to avoid code duplication.
```